src/boundary_operators.jl

```diff
@@ -65,6 +65,19 @@ function convolve_BC_left!{T<:Real,S<:SVector,RBC}(x_temp::AbstractVector{T}, x:
 end
 
 
+function convolve_BC_left!{T<:Real,S<:SVector,RBC}(x_temp::AbstractVector{T}, x::AbstractVector{T}, A::LinearOperator{T,S,:None,RBC})
+    Threads.@threads for i in 1 : A.boundary_point_count
+        @inbounds bc = A.low_boundary_coefs[i]
+        tmp = zero(T)
+        startid = max(0,i-1-div(A.stencil_length, 2))
```

Comment:
For reference that "stash away" is referred to as "hoisting". I think LLVM might hoist it out of the loop, but yes it's better to be safe.